pages/1.13/security/oss/index.md

```diff
@@ -8,15 +8,24 @@ menuWeight: 80
 
 Ensure the network is setup according to the information for [securing your cluster](/1.13/administering-clusters/securing-your-cluster/).
 
-You can enable authentication in your datacenter with DC/OS [oauth](https://github.com/dcos/dcos-oauth). Authentication is managed through the DC/OS web interface. The Admin Router enforces access control.
+All access management in DC/OS is done via the DC/OS Identity and Access Manager (IAM). This includes user account management, login as well as authentication token distribution. The IAM provides an HTTP API for managing user accounts in a RESTful fashion.
```

Comment:
Rewrite `This includes user account management, login as well as authentication token distribution.`
with `This includes user account management, login, and authentication token distribution.`